### PR TITLE
getLastSignedInAccount and signout methods

### PR DESCRIPTION
**Pull request by kuba91**
Added possibility to signout as it is required by the guidelines.
Added possibility to get LastSignedInAccount.

### CODE REVIEW

File: android/src/main/java/xyz/luan/games/play/playgames/PlayGamesPlugin.java [hunk 1]
```diff
@@ -88,7 +88,31 @@ public void onMethodCall(MethodCall call, Result result) {
                 pendingOperation = null;
                 throw ex;
             }
-        } else if (call.method.equals("showAchievements")) {
+        } else if(call.method.equals("signOut")){
```
Comment (luanpotter) [inline android/src/main/java/xyz/luan/games/play/playgames/PlayGamesPlugin.java]: please format this file properly, e.g.

```
} else if (...) {
```

and 

```
try {
    ...
} catch (...) {
    ...
}
```

**Comment by kuba91** [comment]
My bad (probably) I made pull request from form/master and where added also ShowAllLeaderboards method - when I got some time I can split it into branches and then make pull request

File: lib/play_games.dart [hunk 2]
```diff
@@ -339,4 +352,30 @@ class PlayGames {
...
+      if (type == SigninResultType.SUCCESS) {
+        result.account = new Account()
+          ..id = map['id']
+          ..displayName = map['displayName']
+          ..email = map['email']
+          ..hiResImageUri = map['hiResImageUri']
+          ..iconImageUri = map['iconImageUri'];
+      } else {
+        result.message = map['message'];
+      }
```
Comment (luanpotter) [inline lib/play_games.dart:370]: you could simplify this if else structure as:
```
if (type == SigninResultType.SUCCESS) {
    ....
} else if (type != SigninResultType.NOT_SIGNED_IN) {
    ....
}
```

File: lib/play_games.dart [hunk 3]
```diff
@@ -339,4 +352,30 @@ class PlayGames {
...
+    }
+    return result;
+  }
+
+  static Future<Void> signOut() async {
+    final Map<dynamic, dynamic> map = await _channel.invokeMethod('signOut');
+
+    return Void();
```
Comment (luanpotter) [inline lib/play_games.dart]: is this really necessary? I assume you can just do:
```
  static Future<Void> signOut() async {
    await _channel.invokeMethod('signOut');
}
```
or (also good, not sure if it works)
```
```
  static Future<Void> signOut() {
    return _channel.invokeMethod('signOut');
}
```

**Review by luanpotter — APPROVED** [review]
LGTM after these final comments, thanks for the contribution!

File: lib/play_games.dart [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import 'dart:async';
+import 'dart:ffi';
```
Comment (luanpotter) [inline lib/play_games.dart]: why is this being added?

Comment (kuba91) [reply lib/play_games.dart]: If was added because of using Void -I cleaned signOut method and it isnt used anymore

**Pull request closed without merging**